app/forms/idv/api_image_upload_form.rb

```diff
@@ -458,14 +458,16 @@ def store_failed_images(client_response, doc_pii_response)
             front_image_fingerprint: failed_front_fingerprint,
             back_image_fingerprint: failed_back_fingerprint,
             doc_auth_success: client_response.doc_auth_success?,
-            selfie_success: client_response.selfie_success,
+            selfie_status: client_response.respond_to?(:selfie_status) ?
+              client_response.selfie_status : :not_processed,
           )
       elsif doc_pii_response && !doc_pii_response.success?
         document_capture_session.store_failed_auth_data(
           front_image_fingerprint: extra_attributes[:front_image_fingerprint],
           back_image_fingerprint: extra_attributes[:back_image_fingerprint],
           doc_auth_success: client_response.doc_auth_success?,
-          selfie_success: client_response.selfie_success,
+          selfie_status: client_response.respond_to?(:selfie_status) ?
+            client_response.selfie_status : :not_processed,
```

Comment:
to help maintain consistency, this could be a private method reused on line 461:

```
...

selfie_status: selfie_status(client_response),

....

def selfie_status(client_response)
  return client_response.selfie_status if client_response.respond_to?(:selfie_status)

  :not_processed
end
```

Reply:
@amirbey I like this idea and took it one further by extracting the helper method into `ApplicationHelper`, so that we can use the method in `ApiImageUploadForm`, `DocumentCaptureSession` and `DocAuth::Response`.